newshour-namedentity-wikipedialink/readme.md

```diff
@@ -1,19 +1,124 @@
+Each file also squishes duplicate entities into one entry and writes how many times that entry appeared.
+The [base-ner.url](https://github.com/clamsproject/aapb-annotations/tree/b5de0d6b48ba9835c9bf6eaacbf46019dcc12203/newshour-namedentity/golds/aapb-collaboration-21
+) file seems to be the url where the transcriptions from the NAMED ENTITY project to be used for this project.   
+
+### What to Annotate
+* `Wikipedia human-searchable link` - to the correct entity described in the videos. 
+### How to Annotate It
+Search the web for the humanly-searchable link to the correct version of this named entity. 
+Paste this into the `.tab` file or into the annotation tool to update the raw annotation. 
+
+### Decisions, Differentiation, and Precision Level during Annotation
+* **Category Disambiguation** - The disambiguation issue of the previous project becomes an even more difficult issue with this project. 
+In the previous project, one could label different references from a named entity text as different categories. eg. "Washington" as an `organization`, `person`, or `location`.
+In this project, once a deduplicated entry of a certain text is linked to one wikipedia link. 
+The tool believes this is the only link that should match that text. 
+This seems to imply an important limitation to one of the tools/processes, or an unfound way to work around this issue.
+
+* **Only One Link Per Text!!** - A bigger issue actually stems from a feature that initially seems convenient. 
+Once an annotation has been linked, any annotations within the same transcript that have the same label and spelling (case-sensitive) will be linked automatically. 
+While this can be useful, that usefulness is impacted by the fact that the same word or phrase can mean different things. 
+For example, "President Bush" and "the Bush administration" can refer to [George H.W. Bush senior](https://en.wikipedia.org/wiki/George_H._W._Bush) or his son [George W. Bush](https://en.wikipedia.org/wiki/George_W._Bush). 
+Since there's no way to give different links to annotations that have the same label and spelling, **one set will invariably be wrong**. As far as the annotator knows, there is no workaround.
+
+* **Lack of Authoritative Mention/Article** - Not everything mentioned in broadcasts has a wikipedia article. 
+In those cases, search the web for other acceptable/wikipedia links, but when no such source exists, the annotator must simply move on.  
+## Data Format and `process.py`
+
+### `raw` data
+`.tab` file where each line is a new instance of a named entity. This seems to be a tab spaced text file.
+* Format:
+```
+anno_id_tag#\tdate_time_of_annotation\tsource_transcript.ann_file\tentity_text\tcategory\tduplicate_count\tWikipedia_link(normal_url)
+```
+* Fields: 
+  * `anno_id_tag#` - entry number
+  * `date_time_of_annotation` - _TODO: Unverified what this refers to_
```

Comment:
@keighrim Do you know what this refers to? Most of them are Nov 10th, with a time between each of about 30s-1min.

Reply:
summoning @wricketts , do you remember what do columns in the raw NEL file mean?

Reply:
It seems like that's just the timestamp? Based on this [code snippet](https://github.com/clamsproject/aapb-annenv-entitylinking/blob/ff9c7b23a1014fc8053c7315fdcfd683111bbe78/code/model.py#L360-L375)

Reply:
Sorry if it may seem obvious but I'm not familiar with this code. 
What is the timestamp of? When the annotation was done?

Reply:
That was my assumption, but I'm not 100% certain. @keighrim does that sound right to you?

Reply:
I think that is the case; 

https://github.com/clamsproject/aapb-annenv-entitylinking/blob/5ed0ad7ac8504f928ca9225a5c8c50f75bb615d3/code/model.py#L456-L462

https://github.com/clamsproject/aapb-annenv-entitylinking/blob/5ed0ad7ac8504f928ca9225a5c8c50f75bb615d3/code/utils.py#L10-L12